CONTRIBUTING.md

```diff
@@ -0,0 +1,45 @@
+# Contributing to News-sentiment 🎉💃
+First off, thank you for taking the time to contribute!
+
+The following is a set of guidelines for contributing to `news-sentiment`.
+
+## Code of conduct 📖
+This project and everyone participating in it is governed by the [UBC Launch Pad Code of Conduct](https://docs.ubclaunchpad.com/handbook/manifesto#code-of-conduct). By participating you are expected to uphold this code. Although the code's language may be directed towards UBC Launch Pad members, all contributers are expected to abide by it, specifically the following policy:
+> Treat everyone with respect - we do not condone any form of discrimination or intolerance
+
+## What should I know before I get started? 💻
+This is a UBC Launch Pad project. [UBC Launch Pad](https://ubclaunchpad.com) is a Software Engineering Club at the University of British Columbia, and as such, all its members are students. Please be considerate of that fact as we are trying to juggle a full course load and maintaining this project.
+
+This project is built using a [`go`](https://golang.org/) backend and a chrome extension as a frontend.
+
+### Backend 🍑
+The backend is written in `go` and uses go modules for dependency management.
+
+#### Routing 🚗
+We use `go`'s rich http support to handle routing. As of the time I'm writing this, we don't use any special libraries here, but we might use something like [`mux`](https://github.com/gorilla/mux).
+
+#### Persistence 🥭
+We use a `MongoDB` database for persistence, accessed from go.
+
+### Frontend 😄
+
+The Frontend is a chrome extension... Ya that's it at the time!
```

Comment:
```suggestion
The frontend is a chrome extension... Ya that's it at the time!
```

Reply:
@MayKikuchi  I meant suggestions like this! This way I can directly commit your suggestions, and you would get even more github green (for the review and for the commit!)

Reply:
ooooh cool cool! Thanks for letting me know! 😄